app/src/main/java/net/gsantner/markor/activity/DocumentEditFragment.java

```diff
@@ -216,44 +218,62 @@ public void onViewCreated(@NonNull View view, Bundle savedInstanceState) {
         // Set initial wrap state
         initDocState();
 
+        final Runnable after = StringUtils.makeDebounced(_hlEditor, 500, () -> {
+            checkTextChangeState();
+            updateUndoRedoIconStates();
+        });
+        _hlEditor.addTextChangedListener(TextWatcherDummy.after((s) -> after.run()));
+    }
+
+    @Override
+    public void onFragmentFirstTimeVisible() {
```

Comment:
Move initial position code into `onFragmentFirstTimeVisible` as we want it to run after the fragment is _displayed_ to the user.